neo/rawio/baserawio.py

```diff
@@ -511,9 +555,11 @@ def get_analogsignal_chunk(self, block_index=0, seg_index=0, i_start=None, i_sto
                 channel_indexes, = np.nonzero(channel_indexes)
 
         if prefer_slice and isinstance(channel_indexes, np.ndarray):
-            # check if channel_indexes are coninuous and transform to slice
-            # this is usefull for memmap or hdf5 where slice make read lazy
-            # contrary to indexes that make a copy (like numpy.take())
+            # Check if channel_indexes are contiguous and transform to slice argument if possible.
+            # This is useful for memmap or hdf5 where providing a slice causes a lazy read,
+            # rather than a list of indexes that make a copy (like numpy.take()).
+            # :TODO: Why would the user ever want to use the slower method? And should the
+            # default by True? Perhaps the name of this argument should reflect copying or not?
```

Comment:
For the default dehavrio I prefer to leave like this to have the same behavior as previous version.
We can change it later for future version.
For the naming I prefer this because it match the spikeinterface kwarg and chaning would make too much work.